testnets/archwaytestnet/chain.json

```diff
@@ -16,10 +16,10 @@
   "fees": {
     "fee_tokens": [
       {
-        "denom": "uconst",
-        "low_gas_price": 0,
-        "average_gas_price": 0.025,
-        "high_gas_price": 0.05
+        "denom": "aconst",
```

Comment:
thanks for the correction. If this really is the case, then can we please correct the assetlist.json--which still stays uconst--as well?